docs/management_api.md

```diff
@@ -15,18 +15,18 @@ Similar as [Inference API](inference_api.md), Management API also provide a [API
 
 ### Register a model
 
-* POST /models
-** model_url - Model archive download url. support:
-*** local model archive (.mar), the file must be directly in model_store folder.
-*** local model directory, the directory must be directly in model_store foler. This option can avoid MMS extracting .mar file to temporary folder, which will improve load time and reduce disk space usage.
-*** HTTP(s) protocol. MMS can download .mar files from internet.
-** model_name - Name of the model. This name will be used as {model_name} in other API as path. If this parameter is not present, modelName in MANIFEST.json will be used.
-** handler - Inference handler entry-point. This value will override handler in MANIFEST.json if present.
-** runtime - Runtime for the model custom service code. This value will override runtime in MANIFEST.json if present. Default PYTHON.
-** batch_size - Inference batch size, default: 1.
-** max_batch_delay - Maximum delay for batch aggregation, default: 100 millisecnonds.
-** initial_worker - Number of initial workers to create, default: 0.",
-** synchronous - Decides whether creation of worker synchronous or not, default: false.
+`POST /models`
+* model_url - Model archive download url. support:
+    * local model archive (.mar), the file must be directly in model_store folder.
+    * local model directory, the directory must be directly in model_store foler. This option can avoid MMS extracting .mar file to temporary folder, which will improve load time and reduce disk space usage.
+    * HTTP(s) protocol. MMS can download .mar files from internet.
+* model_name - Name of the model. This name will be used as {model_name} in other API as path. If this parameter is not present, modelName in MANIFEST.json will be used.
```

Comment:
```suggestion
* model_name - the name of the model; this name will be used as {model_name} in other API as path. If this parameter is not present, modelName in MANIFEST.json will be used.
```
I'm not sure what this part means: "be used as {model_name} in other API as path"

Reply:
The model_name will be part of url in other API, e.g. scale workers: PUT /models/{model_name}